docs/decision_record.md

```diff
@@ -8,9 +8,37 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 [TOC]
 
+## 2024-01-10 Clarifications on the purpose and form of the JSON schema in service-info
+
+### Decision
+
+We made a series of decisions regarding how the JSON-schema should be used to specify the data that a seqcol server will serve.
+
+- we RECOMMEND you produce an openapi.json for your service.
+- you MUST provide a single schema, this MUST include all possible attributes your service may provide (you cannot have a collection with an attribute that is not defined in your schema).
+- we RECOMMEND your schema use property-level refs to point to terms defined by a central, approved seqcol schema.
```

Comment:
I think we should be clearer that, for concept covered by the central, approved seqcol schema, the implementation MUST use the property definitions provided by the specs by either using property-level refs to point to terms defined in the central seqcol schema (recommended) or reproducing the property from the central seqcol schema